app/packs/stylesheets/decidim/_fontface.scss

```diff
@@ -0,0 +1,38 @@
+@font-face {
```

Comment:
Could you fix the _fontface.scss file so it works with webpacker? You can have an example here: https://github.com/OpenSourcePolitics/decidim-reneweu/pull/9/files